src/ExpandObjects/epfilter.f90

```diff
@@ -14426,11 +14428,7 @@ SUBROUTINE CreateNewUnitary
   ENDIF
   CALL AddToObjFld('Economizer Maximum Limit Dry-Bulb Temperature {C}', base + usEconoUpLimitOff,' ')
   CALL AddToObjFld('Economizer Maximum Limit Enthalpy {J/kg}', base + usEconoEnthLimitOff,' ')
-  IF (econoKind .EQ. etFixedDewPointAndDryBulb) THEN
-    CALL AddToObjFld('Economizer Maximum Limit Dewpoint Temperature (C)',base + usEconoUpDewLimitOff, ' ')
-  ELSE
-    CALL AddToObjStr('Economizer Maximum Limit Dewpoint Temperature (C)',' ')
-  ENDIF
+  CALL AddToObjFld('Economizer Maximum Limit Dewpoint Temperature (C)',base + usEconoUpDewLimitOff, ' ')
```

Comment:
Dew point limit should always be used if input, regardless of economizer type (per documentation).